madmom/features/beats.py

```diff
@@ -48,38 +49,47 @@ class RNNBeatProcessor(SequentialProcessor):
 
     """
 
-    def __init__(self, post_processor=average_predictions, **kwargs):
+    def __init__(self, post_processor=average_predictions, online=False,
+                 nn_files=None, **kwargs):
```

Comment:
Yes. Although you're right about which model can be used (only trained specifically for/with this settings), it can be quite useful to select only one or two of the trained models if processing power is limited.